### ISSUE TITLE
Ad Server Plugin Stuck Processing

### ISSUE THREAD

**Issue by RedTapeTV** [DanielnetoDotCom/YouPHPTube#1364]
While trying to set up a Campaign in the Ad Server Plugin, when I click save it just gets stuck at the Processing screen. I checked the logs and nothing is showing. Any help would be great. 
<img width="1218" alt="screen shot 2019-01-06 at 6 51 55 pm" src="https://user-images.githubusercontent.com/45922853/50743369-09bbdb00-11e5-11e9-8dbd-05436a7ebecc.png">

**Comment by DanielnetoDotCom** [comment]
It is really hard to say something if there is no log error.
Try to verify any JavaScript error . Or maybe the Ajax response

**Comment by RedTapeTV** [comment]
Yeah, I do understand that. I do really appreciate your help still. I am checking into that now. In the meantime not sure if this is related at all but I did reboot my server and checked the logs again and see this error. Not sure if it's related or not but thought I would share. 

[07-Jan-2019 00:11:34 UTC] [sqlDAL::readSql] (mysqlnd) Prepare failed: () <br>
SELECT * FROM  plugins WHERE status='active' AND uuid = '55a4fa56-8a30-48d4-a0fb-8aa6b3fuser3' ;
[07-Jan-2019 00:11:34 UTC] PHP Warning:  mysqli::prepare(): Couldn't fetch mysqli in /var/www/html/YouPHPTube/objects/mysql_dal.php on line 102
[07-Jan-2019 00:11:34 UTC] PHP Warning:  sqlDAL::readSql(): Couldn't fetch mysqli in /var/www/html/YouPHPTube/objects/mysql_dal.php on line 103
[07-Jan-2019 00:11:34 UTC] PHP Warning:  sqlDAL::readSql(): Couldn't fetch mysqli in /var/www/html/YouPHPTube/objects/mysql_dal.php on line 103
[07-Jan-2019 00:11:34 UTC] Array
(
    [0] => Array
        (
            [file] => /var/www/html/YouPHPTube/objects/mysql_dal.php
            [line] => 103
            [function] => log_error
            [args] => Array
                (
                    [0] => [sqlDAL::readSql] (mysqlnd) Prepare failed: () <br>
SELECT * FROM  plugins WHERE status='active' AND uuid = 'e06b161c-cbd0-4c1d-a484-71018efa2f35' ;
                )

**Comment by RedTapeTV** [comment]
Not sure if it helps but also seeing this.

[07-Jan-2019 00:18:36 UTC] VMAP select those options: Array
(
    [0] => 1
    [1] =>
